### PR TITLE
SYSTEST-9245-Log user IDs in an array in MF logs instead of multiple lines

### PR DESCRIPTION
**Pull request by Nandana-NNR**

### CODE REVIEW

File: server/src/index.mjs [hunk 1]
```diff
@@ -82,16 +82,16 @@ server.on('upgrade', function upgrade(request, socket, head) {
 });
 
 // Starter user(s)
-logger.info('Adding user 123...'); stateManagement.addUser('123'); userManagement.addUser('123');
-logger.info('Adding user 456...'); stateManagement.addUser('456'); userManagement.addUser('456');
-logger.info('Adding user 789...'); stateManagement.addUser('789'); userManagement.addUser('789');
+stateManagement.addUser('123'); userManagement.addUser('123');
```
Comment (kschrief) [inline server/src/index.mjs:85]: Can you remove these three lines? With the appId changes adding 123, 456, 789 is preventing 123~A, 456~A, and 789~A from being added which are commonly used userIds.

File: server/src/routes/api/user.mjs [hunk 2]
```diff
@@ -23,6 +23,8 @@
 import { v4 as uuidv4 } from 'uuid';
 import * as userManagement from '../../userManagement.mjs';
 import * as stateManagement from '../../stateManagement.mjs';
+import { logger } from '../../../src/logger.mjs';
```
Comment (kschrief) [inline server/src/routes/api/user.mjs:26]: You don't have to go all the way up to root. You can just ../../logger.mjs

File: server/src/stateManagement.mjs [hunk 3]
```diff
@@ -738,6 +742,7 @@ export const testExports={
 }
 export {
   state,
+  defaultUserArray,
```
Comment (kschrief) [inline server/src/stateManagement.mjs:745]: We can just call userManagement.getUsers(). We don't need to keep track of "default" users

File: server/src/routes/api/user.mjs [hunk 4]
```diff
@@ -47,6 +49,7 @@ function addUser(req, res) {
         status: 'SUCCESS',
         userId: userId
       });
+      logger.info(`Adding user ${userId}`);
```
Comment (kschrief) [inline server/src/routes/api/user.mjs:52]: Can we say "Added" instead of "Adding"?

**Review by kschrief — CHANGES_REQUESTED** [review]

**Review by dhillomk — APPROVED** [review]

**Pull request merged**